test/integration/dualstack/dualstack_test.go

```diff
@@ -256,12 +256,14 @@ func TestCreateServiceSingleStackIPv4(t *testing.T) {
 				return
 			}
 			// validate the service was created correctly if it was not expected to fail
+			svcName := svc.Name
 			svc, err = client.CoreV1().Services(metav1.NamespaceDefault).Get(context.TODO(), svc.Name, metav1.GetOptions{})
```

Comment:
the line of "svcName := svc.Name" should be moved to the below of line 260, or delete line 259